students/11th/daseulsong/js/login.js

```diff
@@ -0,0 +1,20 @@
+'user strict';
+
+const loginBox = document.querySelector('.login__main__idPw');
+const id = document.querySelector('.login__idPw--id');
+const pw = document.querySelector('.login__idPw--pw');
+loginBtn = document.querySelector('.login__btn--blue');
+[id, pw].forEach((item) => {
```

Comment:
```suggestion
const loginBtn = document.querySelector('.login__btn--blue');

[id, pw].forEach((item) => {
```
1.  구분되는 요소들 간에 한줄 띄워주세요
2. loginBtn 역시 const로 선언해주는 것이 좋아보입니다. const로 선언하지 않은 특별한 이유가 있을까요?

Reply:
loginBtn 작업 중 변수의 자료형이 누락된 것으로 보입니다. const 추가했으며, 구분되는 요소 간 공백을 넣었습니다.